arrow-core/src/main/java/com/raphtory/arrowcore/implementation/EdgePartitionManager.java

```diff
@@ -246,7 +258,7 @@ protected EdgePartition getPartition(int partId) {
      *
      * @return the edge partition
      */
-    protected EdgePartition getPartitionAndLoad(int partId) {
+    public EdgePartition getPartitionAndLoad(int partId) {
         EdgePartition p = _partitions.get(partId);
         if (p == null) {
             if (_partitions.size() >= N_ARROW_PARTITIONS) {
```

Comment:
//removeLRUPartition(); 
and 
else {
            //touchPartition(p);
        }

not required below?